src/saving/SaveHelper.cs

```diff
@@ -22,9 +22,52 @@ public enum SaveOrder
         FileSystem,
     }
 
+    public static void RemoveExcessAutoSaves()
+    {
+        int maxAutoSaves = Settings.Instance.MaxAutoSaves;
+
+        var autoSaveNames = new List<string>();
+        var allSaveNames = CreateListOfSaves();
+
+        foreach (var save in allSaveNames.AsEnumerable().Reverse())
+        {
+            if (save.StartsWith("auto_save", StringComparison.CurrentCulture))
```

Comment:
This and the print are basically the only different parts when comparing with RemoveExcessQuickSaves.